Tests/ttLib/ttFont_test.py

```diff
@@ -46,3 +47,54 @@ def test_registerCustomTableClassStandardName():
         assert font[TABLETAG].compile(font) == b"\x04\x05\x06"
     finally:
         unregisterCustomTableClass(TABLETAG)
+
+
+def test_gettabletags_method(tmp_path):
```

Comment:
not sure it's worth adding this very verbose test for such a trivial change

Reply:
Np, happy to remove it.  The behavior was different by reader type and I was confirming that I landed on something that worked for both font reads and ttx reads.